framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -156,7 +156,7 @@ public void run() {
       }
       duration = stopwatch;
 
-      if (DrillTestDefaults.JDBC_DRIVER.equals(DrillTestDefaults.Driver.APACHE)) {
+      if (DrillTestDefaults.JDBC_DRIVER.equals("org.apache.drill.jdbc.Driver")) {
```

Comment:
How about we create a constant called APACHE_DRIVER in DrillTestDefaults.java and use that in all comparisons?